source/schemas/types.txt

```diff
@@ -509,3 +509,124 @@ with the ``title`` field set to the embedded object's type name:
             }
       }
    }
+
+.. _geospatial:
+
+Geospatial Data
+---------------
+
+Geospatial data describes points and other data on the earth's surface. The 
+Realm SDKs support the creation of numerous types of geospatial data. However, 
+only persisted GeoJSON Point data can be queried. 
+
+GeoJSON Point
+~~~~~~~~~~~~~
+
+A GeoJSON Point (GeoPoint) is a single location on the Earth's surface. The GeoPoint 
+schema must contain the following fields:
+
+- ``type`` (string), which is always set to "Point"
+
+- ``coordinates`` (array of doubles). The ``coordinates`` array must contain  
+  at least 2 double values, and may contain a third:
+    
+    - The first double is the longitude of the point, between -180 and 180.
+    - The second is the latitude, between -90 and 90.
+    - The optional third value represents the elevation/altitude of the point, 
+      in meters.
+
+To store a GeoPoint in Atlas for later querying with :ref:`Atlas Device Sync <sync>`, 
+the following rules also apply:
+
+- It must be of type ``object``
+- It must be embedded within another type
+- Additional properties are allowed within the ``coordinates`` array, as long as 
```

Comment:
```suggestion
- Additional entries are allowed within the ``coordinates`` array, as long as 
```
[nit] I think "entries" or "items" is probably more accurate here. Properties makes me think of object properties

Reply:
They are object properties, though, aren't they? We can have additional properties on the object. ?

Reply:
I think the confusion is from "IN the coordinates array".

Items in an array (i.e. the lat/lon doubles) are array elements/entries/items.

The sibling fields of `coordinates` are object properties.